packages/graphql/src/fragments/follow.graphql

```diff
@@ -0,0 +1,46 @@
+fragment FollowMetadata on triples {
+  id
+  subject {
+    id
+    label
+  }
+  predicate {
+    id
+    label
+  }
+  object {
+    id
+    label
+  }
+  vault {
+    id
+    totalShares
+    currentSharePrice
+    positions_aggregate(where: $positionsWhere) {
+      aggregate {
+        count
+        sum {
+          shares
+        }
+      }
+    }
+    positions(
```

Comment:
should we be including things with vars in fragments?

Reply:
we can as long as the variable is passed in from the parent query that the fragment is nested within -- we can always create one w/o the variable for more generic use. these are all optional vars too

Reply:
approving -- we can abstract out if we find it constricting.